eng/native/build-commons.sh

```diff
@@ -59,6 +59,19 @@ check_prereqs()
         echo "Please install CMake 3.14.2 or newer from http://www.cmake.org/download/ or https://apt.kitware.com and ensure it is on your path."; exit 1;
     fi
 
+    if [[ "$__HostOS" == "OSX" ]]; then
+        # Check presence of pkg-config on the path
+        command -v pkg-config 2>/dev/null || { echo >&2 "Please install pkg-config before running this script, see https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md"; exit 1; }
+
+        if ! pkg-config openssl ; then
+            # We export the proper PKG_CONFIG_PATH where openssl was installed by Homebrew
+            # It's important to _export_ it since build-commons.sh is sourced by other scripts such as build-native.sh
+            export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
```

Comment:
Presumably this should set the value additively, not as a replacement...

Reply:
I think replacement is correct here. `build-commons.sh` is sourced by other scripts and we really just need openssl. Also, the default directory will be searched anyway, see the pkg-config documentation about the `PKG_CONFIG_PATH` environment variable:
> PKG_CONFIG_PATH
>  A colon-separated  (on  Windows,  semicolon-separated)  list  of
>  directories to search for .pc files.  The default directory will
>  always be searched after searching  the  path;  the  default  is
>  libdir/pkgconfig:datadir/pkgconfig  where  libdir  is the libdir
>  for pkg-config and datadir is the datadir for pkg-config when it
>  was installed.